src/lib/one-graph/cli-netlify-graph.js

```diff
@@ -0,0 +1,269 @@
+const fs = require('fs')
+const path = require('path')
+const process = require('process')
+
+const { GraphQL, InternalConsole, NetlifyGraph } = require('netlify-onegraph-internal')
+
+const { detectServerSettings, error, getFunctionsDir, log, warn } = require('../../utils')
+
+const { printSchema } = GraphQL
+
+const internalConsole = {
+  log,
+  warn,
+  error,
+  debug: console.debug,
```

Comment:
there is a package for debugging

```javascript
const debug = require('debug')

debug('oneGraph')('My message')
```

this will only be executed if called with `DEBUG=* netlify ...` or `DEBUG=oneGraph netlify ....`

With that you get fine grained console logs on different debugs scopes